src/plugins/intel_cpu/src/nodes/paged_attn.cpp

```diff
@@ -165,7 +165,12 @@ void PagedAttention::createPrimitive() {
 
         size_t key_group_size = cpuConfig.keyCacheGroupSize;
         size_t value_group_size = cpuConfig.valueCacheGroupSize;
-        return make_pa_executor(rtPrecision, kCachePrecision, vCachePrecision, key_group_size, value_group_size);
+        return make_pa_executor(rtPrecision,
+                                kCachePrecision,
+                                vCachePrecision,
+                                key_group_size,
+                                value_group_size,
+                                cpuConfig.keyCacheQuantByChannel);
```

Comment:
1. Can we switch PA to by_channel as default mode?
2. ARM impl doesn't support by_channel quantization (at least with vectorized code), so it needs to set to false